### PR TITLE
Adding site_rank to site_visits

### LINKED ISSUE #285: Add optional site_rank column to site_visits table
We're often using a top site list to crawl.

### PR DESCRIPTION
**Pull request by vringar**
Closes #285

### CODE REVIEW

File: automation/DataAggregator/schema.sql [hunk 1]
```diff
@@ -43,7 +44,7 @@ CREATE TABLE IF NOT EXISTS flash_cookies (
 
 /*
 # crawl_history
- */
+*/
```
Comment (nhnt11) [inline automation/DataAggregator/schema.sql]: Why these changes? The asterisks being on the same column is intentional, no?

Comment (englehardt) [reply automation/DataAggregator/schema.sql]: Yes this indenting was intentional

**Comment by englehardt** [comment]
@vringar would you mind to revert the comment closing bracket indent changes? Aside from that everything LGTM. Thank you!
